INSTALL.md

```diff
@@ -1698,7 +1698,7 @@ Configuration Problems
 ### Selecting the correct target
 
 The `./Configure` script tries hard to guess your operating system, but in some
-cases it does not succeed. You will see a message like the following:
+cases, it does not succeed. You will see a message like the following:
```

Comment:
I don't think the comma is necessary here, because `in some cases it does not succeed` reads just fine